### ISSUE TITLE
Should Feed Escape work in Pydantic Models

### ISSUE THREAD

**Issue by killswitch-GUI** [tiangolo/fastapi#3024]
I'm not seeing `\f` affecting the output of pydantic models in my OpenAPI spec as described here: https://fastapi.tiangolo.com/advanced/path-operation-advanced-configuration/ is this a bug I should open or just a limitation?

**Comment by insomnes** [comment]
@killswitch-GUI doc at link, that you have provided, explicitly says:
You can limit the lines used from the docstring of **a path operation function** for OpenAPI.

**Comment by killswitch-GUI** [comment]
@insomnes I clearly asked if this would affect **Pydantic models in my OpenAPI spec**.. you neither answered my question or provided context to the question I'm asking. Also, I'm well aware of what the docs have... I searched over the Pydantic & FastAPI documentation carefully to make sure I wasn't coming here to piss people like @Kludex off... 

Simply put I want to maintain my development docs and keep my linters happy while excluding those nasty bits to my JSON schema like such:

```python
class Item(ModelWeOwn, SomeOtherModel):
    """Some nice model
    \f
    VERY Long internal description, use case etc.

    Args:
        ModelWeOwn (Type): Somthing

    Return:
        (Type): somthing
    """
    name: str
    description: Optional[str] = None
    price: float
    tax: Optional[float] = None
    tags: Set[str] = []


@app.post("/items/", response_model=Item, summary="Create an item")
async def create_item(item: Item):
    """
    Create an item with all the information:

    - **name**: each item must have a name
    - **description**: a long description
    - **price**: required
    - **tax**: if the item doesn't have tax, you can omit this
    - **tags**: a set of unique tag strings for this item
    \f
    :param item: User input.
    """
    return item
```

**Comment by insomnes** [comment]
@killswitch-GUI sorry, for some reason my brain omitted 
> is this a bug I should open or just a limitation?

part when i was answering your question.
Anyway, doc clearly says that it works only for path operation function, and it work as intended.
It is good enhancement proposal and as i can see @Kludex created PR for it.

**Comment by killswitch-GUI** [comment]
@insomnes no issue at all I appreciate your help and attention. and @Kludex LOL.... idk what to say other than add documentation to your PR so more people don't come here to ask questions...

**Comment by Kludex** [comment]
@killswitch-GUI There's no need to close the issue.

**Comment by killswitch-GUI** [comment]
ok @Kludex

**Comment by tiangolo** [comment]
This was handled by @Kludex in https://github.com/tiangolo/fastapi/pull/3032, it will be available in the next version, FastAPI `0.82.0`. :tada:

**Comment by amacfie** [comment]
Note that there is a bug in the implementation but there is a PR to fix it: https://github.com/tiangolo/fastapi/pull/6039
